src/gui/usbipd_gui.rs

```diff
@@ -171,11 +171,30 @@ impl UsbipdGui {
                     .clone()
                     .unwrap_or_else(|| "Unknown Device".to_string()),
             );
-
             if device.is_bound() {
-                let menu_item = self
+                let menu_item;
+
+                if description.len() > 16 {
+                    let mut short_description = description.clone();
+                    short_description.truncate(16);
```

Comment:
You can avoid the if-else here right away and just always call truncate on the string, since truncate does nothing if the string is already shorter,  (see rust doc about truncate). This should simplify the code and reduce nesting a bit, making everything cleaner.

Reply:
Oh I see now why you did it like this. Fine by me, keep it as is.